### ISSUE TITLE
this shit sucks

### ISSUE THREAD

**Issue by Nicky20228** [computernewb/collabvm-1.2.ts#5]
and thats facts - berner420

**Comment by elijahr2411** [comment]
yeah I'm not gonna take that from someone whose only non-fork repository is this

![image](https://github.com/computernewb/collabvm-1.2.ts/assets/62162399/031fc949-cb11-4a18-b77f-5928c233bcc5)

**Comment by mvoolt** [comment]
$${\color{red}SUPER}$$
# $${\color{red}F}$$
